modules/references/client/components/read-references/Reference.js

```diff
@@ -81,37 +97,25 @@ export default function Reference({ reference, inRecipientProfile }) {
         Math.round((Date.now() - date.getTime()) / 3600 / 24 / 1000),
     );
 
-  const inCreatorProfile = !inRecipientProfile;
-
   return (
     <div className="panel panel-default" id={_id}>
-      <div className="panel-body reference">
-        <ReferenceHeading>
-          {inCreatorProfile && <div>{t('their reply')}</div>}
+      <div className="panel-body">
+        <Header>
```

Comment:
Contemplated encapsulating contents of the header into its own component and re-using it across both experiences (original & response).

They ended up looking quite different at the end so opted in to not doing so.